.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,39 @@
+name: CIFuzz
+on: 
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'  # Run at 00:00 every Sunday
```

Comment:
is this good @shukitchan

Reply:
I think it is still run on "pull_request".

Reply:
in addition to the added schedule